charts/manifests/templates/manifests/templates/_containerTpl.tpl

```diff
@@ -51,6 +51,9 @@ env: {{- include "lib.utils.extras.environment" $context | nindent 2 }}
         {{- include "lib.utils.strings.template" (dict "value" $filteredList "context" $context) | nindent 2 }}
       {{- end }}
     {{- end }}
+    {{- with $values.envFrom }}
+envFrom: {{- include "lib.utils.strings.template" (dict "value" $values.envFrom "context" $context) | nindent 2 }}
+    {{- end }}
```

Comment:
```suggestion
    {{- with $values.envFrom }}
envFrom: {{- include "lib.utils.strings.template" (dict "value" . "context" $context) | nindent 2 }}
    {{- end }}
```

The `with` operator changes the current root scope to the given map. Therefor `$values.envFrom` becomes `.` in that code block. The code still works, because you are referencing variables, which are accessible outside of the with-scope, but with is mainly used to prevent redundancy by writing twice the Value path.